private_gpt/settings/settings.py

```diff
@@ -260,6 +261,21 @@ class OpenAISettings(BaseModel):
         description="OpenAI embedding Model to use. Example: 'text-embedding-3-large'.",
     )
 
+class FireWorksSettings(BaseModel):
+    api_key: str
+    model: str = Field(
+        "accounts/fireworks/models/llama-v3p1-70b-instruct",
+        description="FireWorks Model to use. Example: 'accounts/fireworks/models/llama-v3p1-70b-instruct'.",
+    )
+    # embedding_api_base: str = Field(
+    #     None,
+    #     description="Base URL of OpenAI API. Example: 'https://api.openai.com/v1'.",
+    # )
+    embedding_api_key: str
+    # embedding_model: str = Field(
+    #     "text-embedding-ada-002",
+    #     description="OpenAI embedding Model to use. Example: 'text-embedding-3-large'.",
+    # )
```

Comment:
Does it make sense to have two URLs and API Keys like Ollama? Or will it always run in a non-private environment?

Reply:
using fireworks by default embedded model so had not given option to change it